src/plugins/correlations/common/types.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { SavedObjectAttributes, SavedObjectReference } from 'opensearch-dashboards/server';
+
+export interface CorrelationSavedObjectAttributes extends SavedObjectAttributes {
+  title: string;
+  description: string;
+  correlationType: string;
```

Comment:
we need some basic convention here. Something like `correlation-pluginname/featurename`. Again this can be added in the comments as a full fledged example.